addon/components/sortable-item.js

```diff
@@ -106,11 +104,9 @@ export default class SortableItemComponent extends Component {
       return;
     }
 
-    // let handle = get(this, 'handle');
-
-    // if (handle && !ev.target.closest(handle)) {
-    //   return;
-    // }
+    if (this.args.handle && !ev.target.closest(this.args.handle)) {
```

Comment:
drag handle support. so that dragging will happen only when dragging from a particular section of the column.